requirements.txt

```diff
@@ -24,6 +24,7 @@ pydocx==0.7.0
 Pillow==2.8.2
 
 # IPython
+ipdb==0.10.2
```

Comment:
I'm just trying to preserve all the dependencies we would normally use, it would be disappointing if we get it into prod and there's related module we don't know about which isn't present. But see why you we might want to be more direct.